coderd/database/models.go

```diff
@@ -34,6 +34,26 @@ func (e *AuditAction) Scan(src interface{}) error {
 	return nil
 }
 
+type BuildReason string
+
+const (
+	BuildReasonMember    BuildReason = "member"
```

Comment:
`BuildReasonInitiator` could be clearer!

Reply:
Done!